DOM_creation/lesson1/main.js

```diff
@@ -1,14 +1,5 @@
-// Get list-item id element
-const listItem = document.getElementById("list-item");
-
-//create <li> elment
+// Lesson 1
```

Comment:
same above reason

Reply:
Thank you for your comment Morita-san!

As per your feedback, deleted the comment line(s).
Kindly check the latest commit 
5c9ba1f9a6e5a93bc0d93f44d1688e1bfbe43b68